lib/internal/process/esm_loader.js

```diff
@@ -24,6 +24,9 @@ exports.initializeImportMetaObject = function(wrap, meta) {
 
 exports.importModuleDynamicallyCallback = async function(wrap, specifier) {
   assert(calledInitialize === true || !userLoader);
+  process.emitWarning(
+    'The ESM module loader is experimental.',
+    'ExperimentalWarning', undefined);
```

Comment:
If `calledInitialize` is true we must not emit the warning again. We must also set `calledInitialize` to true after this first warning so we do not warn on every dynamic import.

Reply:
I think I could `if (!calledInitialize) { ...emitWarning }`, what do you think?

Reply:
Yes, plus setting it to true.